app/src/org/commcare/preferences/CommCareServerPreferences.java

```diff
@@ -96,6 +96,10 @@ public static String getSupportEmailAddress() {
         return getServerProperty(PREFS_SUPPORT_ADDRESS_KEY, CommCareApplication.instance().getString(R.string.support_email_address_default)) ;
     }
 
+    public static String getFormRecordKey() {
```

Comment:
I will change the nomenclature around it. Though today our prefs get methods are arranged by the screen they belong to. So since remote form payload url pref is available on ServerPreferences I think the getter should be defined here as well.